frontend/src/components/assessments/assessment-status/EditStatusPopover.tsx

```diff
@@ -73,6 +74,12 @@ const EditStatusPopover = ({
                 closePopover={onClose}
               />
             )}
+            {assessmentStatus !== Status.ARCHIVED && (
```

Comment:
can you refactor the duplicate button to go in this block? it is also shown when status != archived. you can also do a ternary condition for the == archived and != archived